### PR TITLE
Expands TRAC acronym and tidies up explanatory text

### PR DESCRIPTION
**Pull request by sallain**
Connected to archivematica/issues#12

### CODE REVIEW

File: getting-started/other-resources/trac.rst [hunk 1]
```diff
@@ -4,10 +4,12 @@
...
 
-Drupal TRAC review tool
------------------------
+The Trustworthy Repositories Audit & Certification (TRAC) review tool is a Drupal
+implementation of the TRAC checklist, an auditing tool to assess the
+reliability, commitment and readiness of institutions to assume long-term
+preservation responsibilities.
 
-Developed by MIT in a project led by Nancy McGovern, Head of
+The review tool was developed by MIT in a project led by Nancy McGovern, Head of
```
Comment (ross-spencer) [inline getting-started/other-resources/trac.rst]: Hi @sallain there are two lines in this paragraph going over 80 characters. If we can correct those that will be good.

**Review by ross-spencer — CHANGES_REQUESTED** [review]
Hi @sallain. In the introduced changes lines 7, 14, and 15 extend over 80 characters. If these can be fixed that will be good. I notice also in the main body, a few lines with hyperlinks that extend over 80 characters too. I wonder if there is an opportunity, whether you can fix this to [reference style-links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) while you're in the doc to bring the whole page up to date with the style guide. No worries if not. The line changes in the newly introduced text are probably more important.The text itself looks great.

**Comment by sallain** [comment]
@ross-spencer Working on adding reference-style links right now! You're a mind reader.

**Comment by sallain** [comment]
@ross-spencer I've fixed those line lengths and redone all the links. Should be compliant with our style guide now.

**Review by ross-spencer — APPROVED** [review]

**Pull request merged**